tests/unit/Infrastructure/Manage/Dto/EntityDiffTest.php

```diff
@@ -71,6 +71,52 @@ public function test_recursion()
         $this->assertSame([1 => 'is for duck'], $diff['d']);
     }
 
+
+    public function test_grants_and_redirect_urls_are_handled_their_own_special_way()
+    {
+        // No change should result in the redirectUrls not showing up
+        $data = ['metaDataFields.redirectUrls' => ['https://uri1.example.com']];
```

Comment:
I understand that the arrays can be nested and iterated recursively. I don't see any reflection of this in the test.

Reply:
The new tests and the `test_recursion` test go into this feature. In retrospect. We probably never actually needed the recursive nature in the Diff feature. As all array typed metadata options are not to be PUT in a diffed manner. But rather in the 'provide all array values' way..